plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/before/test/impl/SetOpenAssociatedPerspectiveExt.java

```diff
@@ -101,4 +101,9 @@ public boolean hasToRun() {
 		return true;
 	}
 
+	@Override
+	public long getPriority() {
+		return 10000;
```

Comment:
All constants should be be defined as constants (public static final). Also update the JavaDoc so that it is clear what's the priority.